### PR TITLE
Fix #2103-Added option to allow user to choose to delete photos perma…

### LINKED ISSUE #2103: Delete photos feature enhancement.
**Actual Behaviour**

On deleting photos, the photos are moved to the Trashbin and the user is supposed to restore those photos or permanently delete them from the bin.

**Expected Behaviour**

There should be a checkbox option in the delete dialog for moving the photo to trashbin or deleting it permanently(directly from the folder).

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sauravvishal8797**
…nently or move to trashbin.

Fixed #2103 

Changes: A checkbox has been added in the delete dialog through which the user can choose to either delete the image permanently from the storage or to move it to the trashbin.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -1981,45 +1981,53 @@ protected Boolean doInBackground(String... arg0) {
                             if (editMode) {
                                 if (!all_photos && !fav_photos) {
                                     //clearSelectedPhotos();
-                                    succ = addToTrash();
-                                    if(succ){
-                                        addTrashObjectsToRealm(getAlbum().getSelectedMedia());
+                                    if(AlertDialogsHelper.check){
+                                        succ = addToTrash();
+                                        if(succ){
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: space

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 2]
```diff
@@ -1981,45 +1981,53 @@ protected Boolean doInBackground(String... arg0) {
...
-                                            Uri deleteUri = ContentUris
-                                                    .withAppendedId(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, id);
-                                            contentResolver.delete(deleteUri, null, null);
-                                            succ = true;
-                                        } else {
-                                            succ = false;
-                                            // File not found in media store DB
+                                    if(AlertDialogsHelper.check){
+                                        succ = addToTrash();
+                                        if(succ){
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: space

Comment (batbrain7) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: similarly these are present in all the changes

**Comment by sauravvishal8797** [comment]
@batbrain7 Made the changes, please review.

**Pull request merged**